remote_conf.js

```diff
@@ -1,4 +1,17 @@
 module.exports = {
     hub_ip : '',
-    app_id : 'amzn1.echo-sdk-ams.app.999999-d0ed-9999-ad00-999999d00ebe'
+    app_id : 'amzn1.echo-sdk-ams.app.999999-d0ed-9999-ad00-999999d00ebe'//,
+//    Optional channel configuration
+//    channels : [
+//         {
+//           utterance_name : 'watch NBC',
```

Comment:
Would be nice to be able to provide expandable utterance_name 
i.e. "watch {NGC|National Geographics}"
this generates 
WatchTivoWatch{NGC|National Geographics}	to watch NGC
WatchTivoWatch{NGC|National Geographics}	 watch NGC
WatchTivoWatch{NGC|National Geographics}	to watch National Geographics
WatchTivoWatch{NGC|National Geographics}	 watch National Geographics

which is not accepted by amazon .